### PR TITLE
Added Exception Handling and Logging

### PR DESCRIPTION
**Pull request by sritambehera**
1. Changed phone no field to string from long
2. Added Exception Handling
3. Added Logging

### CODE REVIEW

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java [hunk 1]
```diff
@@ -15,49 +19,57 @@
...
+		log.info("Post Controller Started");
+		return new ResponseEntity<>(service.addShipper(postShipper),HttpStatus.CREATED);
 	}
-	
+
 	@GetMapping("/shipper")
-	public List<Shipper> getShippers(@RequestParam(required = false) Boolean companyApproved, @RequestParam(required = false) Integer pageNo) {
-		return service.getShippers(companyApproved, pageNo);
-		
+	public ResponseEntity<List<Shipper>> getShippers(
```
Comment (badireddisatyasaivarun) [inline ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:46]: should throw EntityNotFoundException for get, put, delete

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java [hunk 2]
```diff
@@ -15,49 +19,57 @@
...
+		return new ResponseEntity<>(service.getShippers(companyApproved, pageNo),HttpStatus.OK);
+
 	}
-	
+
 	@GetMapping("/shipper/{shipperId}")
-	private Shipper getOneShipper(@PathVariable String shipperId) {
-		return service.getOneShipper(shipperId);
+	private ResponseEntity<Object> getOneShipper(@PathVariable String shipperId) {
+		log.info("Get Controller Started");
```
Comment (badireddisatyasaivarun) [inline ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:56]: get method by id, be specific

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Model/UpdateShipper.java [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 @Data
 public class UpdateShipper {
-	private Long phoneNo;
+	private String phoneNo;
```
Comment (badireddisatyasaivarun) [inline ShipperAPI/src/main/java/com/springboot/ShipperAPI/Model/UpdateShipper.java:7]: we cannot update phoneNo in shipper..

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java [hunk 4]
```diff
@@ -15,49 +19,57 @@
...
-	
+
+
 	@DeleteMapping("/shipper/{shipperId}")
-	public ShipperDeleteResponse deleteShipper(@PathVariable String shipperId){
-		return service.deleteShipper(shipperId);
+	public ResponseEntity<Object> deleteShipper(@PathVariable String shipperId){
+		log.info("Delete Controller Started");
+		service.deleteShipper(shipperId);
+		return new ResponseEntity<>("Successfully deleted",HttpStatus.OK);
```
Comment (badireddisatyasaivarun) [inline ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:72]: why the response is predefined

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Model/UpdateShipper.java (hunk 3 above)
Comment (sritambehera) [reply ShipperAPI/src/main/java/com/springboot/ShipperAPI/Model/UpdateShipper.java:7]: Yes, added exception for that.

File: ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java (hunk 2 above)
Comment (sritambehera) [reply ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:56]: Why should we do that ? I am not getting it. Can you tell me the purpose of adding user id to the console ?

Comment (badireddisatyasaivarun) [reply ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:56]: I am not telling to add UserId, but just asking you to specify  it came from "get method by Id"

Comment (sritambehera) [reply ShipperAPI/src/main/java/com/springboot/ShipperAPI/Controller/ShipperController.java:56]: oh, ok.

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**